packages/core/src/Search.cpp

```diff
@@ -24,6 +24,16 @@
 
 #include <cstdlib>
 
+#ifdef WIN32
+#define ENV_LIB_PATH "CMAKE_PREFIX_PATH"
+#define LIB_PATH "lib/soss"
+#define IS_NOT_ABSOLUTE_PATH(path) path.find("C:\\") == std::string::npos && path.find("c:\\") == std::string::npos
```

Comment:
Could we consider using [`std::filesystem::path::is_absolute(~)`](https://en.cppreference.com/w/cpp/filesystem/path/is_absrel) at least for the Windows version?

It's a bit harder to get C++17 support on 18.04 (and not as important because the test for Linux is trivial), but I suspect it should already be available on whatever the latest version of MSVC is.

Reply:
Thanks @mxgrey for this approach. We work with CI in Windows with Dockers and, at the moment, only have VS2017 installed there. Our plans are to update to VS2019 but we are having troubles installing it on Docker. But we consider this for the future. Thanks!